src/NServiceBus.Core/Routing/RoutingSettings.cs

```diff
@@ -72,7 +72,7 @@ public void RouteToEndpoint(Assembly assembly, string @namespace, string destina
         /// </summary>
         public void DoNotEnforceBestPractices()
         {
-            Settings.Set(RoutingFeature.EnforceBestPracticesSettingsKey, false);
+            Settings.Set(RoutingComponent.EnforceBestPracticesSettingsKey, false);
```

Comment:
Not an issue to hold off this PR. Just a food for thought: Since Routing is now a core component could the core component exists early enough so that we could directly set the flag on the core component?